playbooks/tags.json

```diff
@@ -0,0 +1,6 @@
+[
+    "PostCreate",
+    "system",
```

Comment:
Why these are marked as system?

Reply:
These playbooks were already marked as System. Do this tag need to be removed?